cdap-docs/application-templates/source/etl/templates/sinks/real-time/cube.rst

```diff
@@ -16,3 +18,34 @@ sink.
 
 To configure transformation from a StructuredRecord to a CubeFact, the mapping
 configuration is required, following StructuredRecordToCubeFact documentation.
```

Comment:
It would be something like:
```
 following `StructuredRecordToCubeFact documentation. 
<http://docs-cask.co/cdap/current/en/reference-manual/javadocs/co/cask/cdap/api/data/format/StructuredRecord.html>`__
```
though I don't see (even in 3.1-SNAPSHOT) any appropriate docs.